janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java

```diff
@@ -70,12 +70,18 @@ public void testReadWideVertexWithManyProperties() throws Exception {
         Map<String, Object> propertiesOnVertex = graph.traversal().V().valueMap().next();
         List<?> valuesOnP = (List)propertiesOnVertex.values().iterator().next();
         assertEquals(numProps, valuesOnP.size());
+        for (int i=0;i< numProps; i++) {
+          assertEquals(Integer.toString(i), valuesOnP.get(i).toString() );
```

Comment:
Please delete extra space before closing parenthesis.

Reply:
I will fix it too. thx